packages/react/src/components/login/components/LoginCallbackHandler.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { User } from 'oidc-client-ts';
```

Comment:
I am getting this mysterious error from this import: `'oidc-client-ts' should be listed in the project's dependencies. Run 'npm i -S oidc-client-ts' to add iteslint[import/no-extraneous-dependencies](https://github.com/import-js/eslint-plugin-import/blob/v2.28.0/docs/rules/no-extraneous-dependencies.md)`

Reply:
This is weird. The package is listed in package.json and this error does not show in my local env.  And the project is built here just fine. Have you ran yarn install when switching branches? Your IDE seems to warn about all new packages, so that might be it.

Reply:
I'm sure I did, even ran it multiple times but could've been VSC messing with me. I'll try again.

Reply:
https://github.com/City-of-Helsinki/helsinki-design-system/pull/1042#discussion_r1363685512